src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -393,10 +393,11 @@ TEST_F(PolicyHandlerTest, AppServiceUpdate_CheckAppService) {
   ChangePolicyManagerToMock();
   std::string file_name("sdl_pt_update.json");
   std::ifstream ifile(file_name);
-  Json::Reader reader;
+  Json::CharReaderBuilder builder;
   std::string json;
   Json::Value root(Json::objectValue);
-  if (ifile.is_open() && reader.parse(ifile, root, true)) {
+  if (ifile.is_open() &&
```

Comment:
@ydementieiev Are you sure that you need to perform all further actions in case when ifile.is_open() returns `false` ?

Reply:
@AByzhynar  this behavior was in tests before my changes and I decided not to change it as part of this pull request